visualization/app/codeCharta/ui/nodeContextMenu/nodeContextMenu.po.ts

```diff
@@ -8,4 +9,9 @@ export class NodeContextMenuPageObject {
 			visible: true
 		})
 	}
+
+	public async exclude() {
+		this.page.click("#exclude-button")
+		await delay(500)
```

Comment:
How do we want to achieve that though?
Running a for-loop for X amount of ms that checks rapidly if the element was removed from the tree-view?

Reply:
You could check it once per ms. That's off course not an ideal way though and we'd have to have a maximum time. There are normally better ways like checking for a specific event though.

This would nevertheless work:

```js
while (true) {
  await delay(1)
  if (conditionToWaitFor) {
    return // Or break, depending on the context
  }
}
```